mattermost-10.0.yaml

```diff
@@ -97,7 +94,7 @@ update:
   github:
     identifier: mattermost/mattermost
     strip-prefix: v
-    tag-filter: v9.
+    tag-filter: v10.0
```

Comment:
should we enable minor version updates here? Using `v10.` we will get update for v10.1.0 as well. Upstream is also creating some `rc` releases which can skip here.

Reply:
Good questions - as we don't have 'use-tag:true' set here, it defaults to the GitHub releases. We filter out non-GA releases here automatically when polling releases instead of tags.

v10.0 is the version stream: https://endoflife.date/mattermost. So when v10.0.1 and v10.0.2 come along, this will auto-increment. if v10.1.0 comes along, thats a new version stream, and isn't built by this melange. Our automation will take care of creating a new melange for that version stream

Reply:
Ahh missed the filename :( 

this looks good. Thanks Mark.